spec/controllers/application_controller_spec.rb

```diff
@@ -1459,6 +1459,15 @@ def index
       end
     end
 
+    context "for a Maryland return" do
+      let(:state_code) { "md" }
+
+      it "returns the NY withdrawal date deadline" do
+        expect(subject.withdrawal_date_deadline(state_code)).to eq Rails.configuration.state_file_withdrawal_date_deadline_md
+      end
+    end
+
+
     context "for an Arizona return" do
```

Comment:
should this say it "returns the MD withdrawal date"?

Reply:
yup